src/lib/cli.js

```diff
@@ -8,20 +8,20 @@
 
 import path from 'path';
 import semver from 'semver';
-import chalk from 'chalk';
+const { bgYellow, bgRed } = require('kleur');
```

Comment:
I'd preffer here `import 'kleur';` .. unless they do not follow ES modules.

Reply:
Same in other places.

Reply:
Ok, I'll change it.

Reply:
The reason is https://github.com/verdaccio/eslint-config-verdaccio is more aggressive with linting, after migrating to Typescript it only accepts import (and only require in some specific scenarios)